packages/foreman/nodejs-theforeman-builder/nodejs-theforeman-builder.spec

```diff
@@ -177,114 +181,118 @@ BuildArch: noarch
+Provides: bundled(npm(@babel/plugin-proposal-async-generator-functions)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-proposal-class-properties)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-proposal-dynamic-import)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-proposal-json-strings)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-proposal-nullish-coalescing-operator)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-proposal-numeric-separator)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-proposal-object-rest-spread)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-proposal-optional-catch-binding)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-proposal-optional-chaining)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-proposal-private-methods)) = 7.10.1
+Provides: bundled(npm(@babel/plugin-proposal-unicode-property-regex)) = 7.10.1
 Provides: bundled(npm(@babel/plugin-syntax-async-generators)) = 7.8.4
+Provides: bundled(npm(@babel/plugin-syntax-class-properties)) = 7.10.1
 Provides: bundled(npm(@babel/plugin-syntax-dynamic-import)) = 7.8.3
 Provides: bundled(npm(@babel/plugin-syntax-json-strings)) = 7.8.3
-Provides: bundled(npm(@babel/plugin-syntax-jsx)) = 7.8.3
+Provides: bundled(npm(@babel/plugin-syntax-jsx)) = 7.10.1
 Provides: bundled(npm(@babel/plugin-syntax-nullish-coalescing-operator)) = 7.8.3
-Provides: bundled(npm(@babel/plugin-syntax-numeric-separator)) = 7.8.3
+Provides: bundled(npm(@babel/plugin-syntax-numeric-separator)) = 7.10.1
 Provides: bundled(npm(@babel/plugin-syntax-object-rest-spread)) = 7.8.3
 Provides: bundled(npm(@babel/plugin-syntax-optional-catch-binding)) = 7.8.3
 Provides: bundled(npm(@babel/plugin-syntax-optional-chaining)) = 7.8.3
-Provides: bundled(npm(@babel/plugin-syntax-top-level-await)) = 7.8.3
-Provides: bundled(npm(@babel/plugin-transform-arrow-functions)) = 7.8.3
-Provides: bundled(npm(@babel/plugin-transform-async-to-generator)) = 7.8.3
-Provides: bundled(npm(@babel/plugin-transform-block-scoped-functions)) = 7.8.3
-Provides: bundled(npm(@babel/plugin-transform-block-scoping)) = 7.8.3
-Provides: bundled(npm(@babel/plugin-transform-classes)) = 7.9.2
-Provides: bundled(npm(@babel/plugin-transform-computed-properties)) = 7.8.3
-Provides: bundled(npm(@babel/plugin-transform-destructuring)) = 7.8.8
-Provides: bundled(npm(@babel/plugin-transform-dotall-regex)) = 7.8.3
-Provides: bundled(npm(@babel/plugin-transform-duplicate-keys)) = 7.8.3
-Provides: bundled(npm(@babel/plugin-transform-exponentiation-operator)) = 7.8.3
-Provides: bundled(npm(@babel/plugin-transform-for-of)) = 7.9.0
-Provides: bundled(npm(@babel/plugin-transform-function-name)) = 7.8.3
-Provides: bundled(npm(@babel/plugin-transform-literals)) = 7.8.3
-Provides: bundled(npm(@babel/plugin-transform-member-expression-literals)) = 7.8.3
-Provides: bundled(npm(@babel/plugin-transform-modules-amd)) = 7.9.0
-Provides: bundled(npm(@babel/plugin-transform-modules-commonjs)) = 7.9.0
-Provides: bundled(npm(@babel/plugin-transform-modules-systemjs)) = 7.9.0
```

Comment:
We have those pinned in foreman-core because of an issue with babel-plugin-dynamic-import-node, will this not cause issues?